tests/starship/README.md

```diff
@@ -0,0 +1,10 @@
+# End-To-End Tests
+Multi-chain system tests that run against the demo app.
```

Comment:
It would be very helpful to have some infos about starship a link to the project here and a list of requirements what is needed to run this test.

I got `Get "http://localhost:8081/chains/mesh-osmosis-1": dial tcp [::1]:8081: connect: connection refused` error when I just execute the make test